### ISSUE TITLE
how to show dice score when not use dice loss ?

### ISSUE THREAD

**Issue by 775397252** [qubvel/segmentation_models.pytorch#492]
As the title says.

**Comment by 775397252** [comment]
i create this file :
```
from torch import nn

from segmentation_models_pytorch.base.modules import Activation

from segmentation_models_pytorch.utils import base
from segmentation_models_pytorch.utils import functional as F



class DICE(base.Metric):

    def __init__(self, eps=1e-7, threshold=0.5, activation=None, ignore_channels=None, **kwargs):
        super().__init__(**kwargs)
        self.eps = eps
        self.threshold = threshold
        self.activation = Activation(activation)
        self.ignore_channels = ignore_channels

    def forward(self, y_pr, y_gt, smooth=1e-5, activation='sigmoid'):
        activation_fn = self.activation
        pred = activation_fn(y_pr)
        N = y_gt.size(0)
        pred_flat = pred.view(N, -1)
        gt_flat = y_gt.view(N, -1)
        intersection = (pred_flat * gt_flat).sum(1)
        unionset = pred_flat.sum(1) + gt_flat.sum(1)
        loss = (2 * intersection + smooth) / (unionset + smooth)
        return loss.sum() / N
```
then    
```
metrics = [
        DICE(),
    ]
```
is any error?  can anyone can help me?
